config/cscs.py

```diff
@@ -114,7 +114,8 @@ class ReframeSettings:
                         'descr': 'Hybrid nodes (Haswell/P100)',
                         'max_jobs': 100,
                         'resources': {
-                            'switches': ['--switches={num_switches}']
+                            'switches': ['--switches={num_switches}'],
```

Comment:
Why do you add `switches` again?

Reply:
switches are important for daint, but they won't be present for dom

Reply:
Yes, sorry, ignore my comment.